alephmarcreader/tests/test_Marc21Reader.py

```diff
@@ -28,6 +28,22 @@ def test_recipient(self):
         self.assertEqual(recipient[0].gnd, u'(DE-588)120379260')
         self.assertEqual(recipient[0].roles, [u'rcp'])
 
+        marc21_rd = AlephMarc21Reader('alephmarcreader/tests/sample_data/Marc21/000056870.marc')
+
+        recipient = marc21_rd.get_recipient()
+
+        self.assertEqual(len(recipient), 2)
+
+        self.assertEqual(recipient[0].name, u'Universit\xe4t Basel.')
```

Comment:
please add a check for the type (person or organisation), both for the string constant (`get_type`) an the actual instance (Person, Organisation)

Reply:
actually the same would have to be done for the already existing tests for persons

Reply:
done